packages/math-input/src/components/keypad/keypad-mathquill.stories.tsx

```diff
@@ -75,6 +75,10 @@ export function V2KeypadWithMathquill() {
                             multiplicationDot
                             preAlgebra
                             trigonometry
+                            sendEvent={async (event) => {
+                                // eslint-disable-next-line no-console
+                                console.log("Send Event:", event);
```

Comment:
Might look into [Storybook actions](https://storybook.js.org/docs/react/essentials/actions) here, it's already one of our deps I believe.

Reply:
I considered using `action`, but I thought they didn't completely fit. Actions are meant to show parameters for event handlers and I don't see `sendEvent` as an event handler in the UI sense. I'm probably overthinking it.